virttest/http_server.py

```diff
@@ -93,7 +95,7 @@ def translate_path(self, path):
 
         """
         # abandon query parameters
-        path = urlparse(path)[2]
+        path = urlparse(decode_to_text(path))[2]
```

Comment:
I don't see the necessity for decode_to_text (path)

Reply:
The urlparse will also parse the bytes string.